packages/subgraph/src/plugin/pluginSetupProcessor.ts

```diff
@@ -96,13 +98,13 @@ export function handleInstallationApplied(event: InstallationApplied): void {
     pluginEntity.appliedPluginRepo = pluginPreparationEntity.pluginRepo;
     pluginEntity.appliedVersion = pluginPreparationEntity.pluginVersion;
   }
-  pluginEntity.pluginAddress =  event.params.plugin;
+  pluginEntity.pluginAddress = event.params.plugin;
   pluginEntity.appliedPreparation = preparationId;
   pluginEntity.appliedSetupId = event.params.appliedSetupId;
   pluginEntity.state = 'Installed';
   pluginEntity.save();
 
-  addPlugin(dao, event.params.plugin);
+  // addPlugin(dao, event.params.plugin);
```

Comment:
why commented

Reply:
we moved to prepare installation, as it was not picking up plugin settings
will be fixed in future pr, 
it might need more refactoring